packages/msw-trpc/src/create.ts

```diff
@@ -0,0 +1,44 @@
+import { AnyTRPCRouter } from '@trpc/server'
+
+import { MswTrpc, TRPCMswConfig } from './types.js'
+
+import { trpc } from './handler.js'
+import { HttpHandler } from 'msw'
+
+const createTRPCMsw = <Router extends AnyTRPCRouter>(config: TRPCMswConfig) => {
+  const { links, transformer } = config
+
+  // @ts-expect-error any
+  const createUntypedTRPCMsw = (pathParts: string[] = []) => {
```

Comment:
Just a nit, but I would call it `pathSegments`. 

Parts feels odd

Reply:
https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL parts seems to be a term that is used when talking about url's so I'll resolve for now!